com.chartboost.helium.demo/Assets/Samples/Helium SDK/3.3.1/Facebook/Editor/Optional-HeliumFacebookDependencies.xml

```diff
@@ -8,12 +8,10 @@
         <androidPackage spec="com.facebook.android:audience-network-sdk:6.12.0" />
     </androidPackages>
     <iosPods>
+        <!-- Helium Facebook Adapter -->
+        <iosPod name="ChartboostHeliumAdapterFAN" version="~> 3.3"/>
+
         <!-- FAN iOS SDK -->
-        <iosPod name="ChartboostHeliumAdapterFAN" version="~> 3.3">
-            <sources>
-                <source>https://github.com/CocoaPods/Specs</source>
-            </sources>
-        </iosPod>
-        <iosPod name="FBAudienceNetwork" version="~> 6.12.0" />
+        <iosPod name="FBAudienceNetwork" version="~> 6.12.0"/>
```

Comment:
Think we may go with route. So if there's a 6.12.1 version, it can also be picked up. Perhaps, may want to check what is done in our Helium iOS SDK.

```suggestion
        <iosPod name="FBAudienceNetwork" version="~> 6.12"/>
```